src/test/java/com/googleinterns/zoomtube/servlets/TranscriptServletTest.java

```diff
@@ -69,9 +70,12 @@
       (new LocalDatastoreServiceTestConfig()).setNoStorage(true);
   private static final LocalServiceTestHelper localServiceHelper =
       new LocalServiceTestHelper(datastoreConfig);
-  private static final String LECTURE_ID_A = "123";
-  private static final String LECTURE_ID_B = "345";
-  private static final String LECTURE_ID_C = "234";
+  private static final String LECTURE_ID_A_AS_STRING = "123";
```

Comment:
I would not keep both. You can create the long and then convert it to a string.

Reply:
Look at https://docs.oracle.com/javase/7/docs/api/java/lang/Long.html for converting a Long to a String.

Reply:
Done! I kept the lecture ids as `Long`